### PR TITLE
Readme: Publish default user "core".

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: docs/flatcar-linux/kvm-libvirt.md [hunk 1]
```diff
@@ -186,6 +186,8 @@ ssh-add ~/.ssh/id_rsa
 ssh-add -L
 ```
 
+To login by SSH, use the default user `core` -> [ssh.tf](https://github.com/kinvolk/lokomotive-kubernetes/blob/master/kvm-libvirt/flatcar-linux/kubernetes/ssh.tf#L8)
```
Comment (pothos) [inline docs/flatcar-linux/kvm-libvirt.md]:
```suggestion
If you want to login by SSH later for debugging purposes, use the default user `core`.
```

**Review by pothos — CHANGES_REQUESTED** [review]
Hi,
thank you for your contribution!
I think the link to the terraform file is not necessary but rather might confuse others. The `core` login is the default in Flatcar Container Linux as it comes from the CoreOS Container Linux default.

File: docs/flatcar-linux/kvm-libvirt.md (hunk 1 above)
Comment (pothos) [reply docs/flatcar-linux/kvm-libvirt.md]: Shall I commit the suggestion and go on with merging?

**Comment by ghost** [comment]
Sorry,  give me one day and then you can merge it. Thank you!

Kai Lüke <notifications@github.com> schrieb am Mi., 29. Jan. 2020, 12:41:

File: docs/flatcar-linux/kvm-libvirt.md (hunk 1 above)
Comment (ghost) [reply docs/flatcar-linux/kvm-libvirt.md]: Thanks @pothos . I committed your suggestion.

**Review by pothos — APPROVED** [review]

**Pull request merged**